xc7/utils/prjxray_edge_library.py

```diff
@@ -994,9 +994,8 @@ def get_tile_loc(tile_pkey):
 
 
 def yield_edges(
-        const_connectors, delayless_switch, phy_tile_pkey, src_connector,
```

Comment:
This is not the correct approach, as it is deferring the change too late.  The modification to the switch should be done here: https://github.com/SymbiFlow/symbiflow-arch-defs/blob/master/xc7/utils/prjxray_form_channels.py#L156

If you change it at this point, the edge import code won't need any special code, and the connection database will reflect the new switch.  This also allows the timing values of the switch to be preserved.

Reply:
Done